tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"compilerOptions": {
+		"declaration": true,
+		"emitDeclarationOnly": true,
+		// Ensure that Babel can safely transpile files in the TypeScript project
+		"isolatedModules": true,
+		"lib": ["ES2019"],
+		"module": "CommonJS",
+		"outDir": "./lib",
+		"strict": false,
+		"target": "ES2019"
+	},
+	"include": ["src"],
+	"exclude": ["node_modules", "**/__tests__/*"]
```

Comment:
Good point, I'll try to remember where that came from, and figure out why. It could be the TypeScript reference tsconfig files, in which case we can definitely replace with "tests".